src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -400,7 +400,6 @@ kibana_vars=(
     xpack.securitySolution.prebuiltRulesPackageVersion
     xpack.spaces.maxSpaces
     xpack.task_manager.max_attempts
-    xpack.task_manager.max_poll_inactivity_cycles
```

Comment:
Does this need to go through the deprecation / breaking change process?

Reply:
@jbudz ^^

Reply:
I'll leave it up to the team - not familiar with the setting and whether its use has been advised in any production scenarios.